cmd/adapter/adapter.go

```diff
@@ -246,3 +270,25 @@ func makeHTTPClient(inClusterAuth bool, kubeConfigPath string) (*http.Client, er
 	}
 	return &http.Client{Transport: tr}, nil
 }
+
+func makePrometheusCAClient(caFilename string) (*http.Client, error) {
+	pool, err := x509.SystemCertPool()
```

Comment:
I think [`NewCertPool()`](https://github.com/golang/go/blob/41e62b8c49d21659b48a95216e3062032285250f/src/crypto/x509/cert_pool.go#L21), would suffice here and wouldn't require having a cert in place (which busybox doesn't).